components/dropdown/dropdown-content-mixin.js

```diff
@@ -759,6 +763,10 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 			}
 		}
 
+		if (mobileTrayBottom) {
+			maxWidthOverride = '100vw';
+		}
```

Comment:
Nit: this is getting a little bit unwieldily with all the `if (mobileTrayThis) { .. }` spread across the next ~100 lines or so. There might be an opportunity to refactor it a bit, or maybe move some of it out into helper functions.

It's tricky because it's nice to have things laid out like you do where you declare a variable (e.g. `widthOverride`) and then right after you calculate it with an `if - elseif - else`. But on the flip side it'd be a lot less code to just have `if (mobileTrayBottom) { thing1 = whatever; thing2 = that; } else if (mobileTrayLeft) { thing1 = that; }`.

Anyway, it's understandable how it is... just getting a bit complex and potentially fragile. :)

Reply:
Yep, totally agree. I'll work on refactoring it this afternoon.

Reply:
Looks a lot better now!